tests/catch/CMakeLists.txt

```diff
@@ -46,32 +50,33 @@ message(STATUS "ROCM Path: ${ROCM_PATH}")
 
 
 if(UNIX)
-    set(CMAKE_CXX_COMPILER "${HIP_PATH}/bin/hipcc")
-    set(CMAKE_C_COMPILER "${HIP_PATH}/bin/hipcc")
-    set(HIPCONFIG_EXECUTABLE "${HIP_PATH}/bin/hipconfig")
-    execute_process(COMMAND perl ${HIPCONFIG_EXECUTABLE} --version
-                    OUTPUT_VARIABLE HIP_VERSION
-                    OUTPUT_STRIP_TRAILING_WHITESPACE)
+    set(CMAKE_CXX_COMPILER "${HIP_PATH}/bin/hipcc.bin")
```

Comment:
Currently usage of hipcc.bin is not enabled by default. You can use an env var HIP_USE_PERL_SCRIPTS=0 to enable hipcc.bin usage through hipcc.

Reply:
Even when using Perl-based hipcc, I use `hipcc ./file.cc ....` not `perl hipcc ./file.cc ...` I figured this is more universal?